app/lib/dartdoc/backend.dart

```diff
@@ -262,6 +274,25 @@ class DartdocBackend {
           () async => retry<FileInfo?>(
             () async {
               try {
+                if (uploadedFilePaths.contains(relativePath)) {
+                  final info = await _storage.info(objectName);
+                  return FileInfo(lastModified: info.updated, etag: info.etag);
+                }
```

Comment:
It seems messy that we mix these two kinds of files in the same handler... Why do need this?